docs/source/pre_application/cloud_node_management.rst

```diff
@@ -79,71 +28,64 @@ Now NuCypher Ops is installed we can create a droplet on Digital Ocean:
+    Backup data can be found here: /SOME_PATH//nucypher-ops/configs/mainnet/nucypher/remote_worker_backups/
 
 This tells us the location of several config files and helpfully prints the IP address of our newly created node (you can also see this on the Digital Ocean dashboard).
 Let's ``ssh`` into it and look at the logs:
 
 .. code-block:: bash
 
     $ ssh root@123.456.789.xxx
-    root@nucypher-ibex-1:~#
-    root@nucypher-ibex-1:~# sudo docker logs --follow ursula
+    root@nucypher-mainnet-1:~#
+    root@nucypher-mainnet-1:~# sudo docker logs --follow ursula
     ...
     ! Operator 0x06E11400xxxxxxxxxxxxxxxxxxxxxxxxxxxx1Fc0 is not funded with ETH
     ! Operator 0x06E11400xxxxxxxxxxxxxxxxxxxxxxxxxxxx1Fc0 is not bonded to a staking provider
     ...
 
-These lines will print repeatedly until the Operator is funded with some rinkeby ETH and bonded to a staking provider.
-Send rinkeby ETH to the operator address that is printed
-
+These lines will print repeatedly until the Operator is funded with some mainnet ETH and bonded to a staking provider.
+Send mainnet ETH to the operator address that is printed.
 
-Stake and Bond
---------------
-Now that our operator is funded with ETH, we're ready to stake and bond.
-At this point you need some testnet ETH and 40,000 T in a metamask wallet.
-Again, ask in the discord if you need help with this.
 
-Navigate to the `Testnet Staking Dashboard <https://dn3gsazzaajb.cloudfront.net/manage/stake>`_ and connect your metamask wallet.
-Go to the **stake** tab and click "Stake liquid T on rinkeby"
-
-.. image:: ../.static/img/testnet_stake_dashboard.png
-    :target: ../.static/img/testnet_stake_dashboard.png
+Stake an Bond
+-------------
 
-Allow the 40,000 T spend, and then stake it.
-Both transactions will require authorization via metamask.
-You can ignore the **Configure Addresses** option - they should all default to the currently connected account.
+If you have not already done so you'll need to establish a stake on the threshold
+staking dashboard ``https://dashboard.threshold.network/overview/network``.
+After you've established your stake, proceed to the PRE node bonding dashboard to bond your node's
+operator address to your stake. ``https://stake.nucypher.network/manage/operator``.
 
-Once those transactions are confirmed, swith to the **bond** tab.
-Here you will paste the Operator address that is being printed by the docker logs:
 
-.. image:: ../.static/img/testnet_bond_dashboard.png
-    :target: ../.static/img/testnet_bond_dashboard.png
+Monitor Remote Node
+-------------------
 
-Once that transaction is confirmed, switch back to view the logs of the node.
-You should see:
+Send a small amount of ETH to your operator address so it can perform the initial start transaction which signals that your
+node is open for business. Once you've funded and your staking transaction is confirmed, view the logs of the node.
+It will automatically detect that you have staked.  You should see:
```

Comment:
```suggestion
Send a small amount of ETH to your Operator address so it can perform the initial confirmation transaction which signals that your
node is open for business. Once you've funded the Operator address and bonded to the stake, view the node's logs.
It will automatically detect both completed actions.  You should see:
```

Reply:
✔️